### ISSUE TITLE
Search is not working after user changed the endpoint and Submit same call again 

### ISSUE THREAD

**Issue by sweetw0r** [mmattozzi/cocoa-rest-client#77]
1. Open the app
2. Send a request 
3. Go to respect and press CMD + F
4. Perfume some search to make sure it is working 
5. Change the endpoint and submit the call e.g. in my case and I replace one ID to another (35e6d7e5-51a1-47da-ae6e-581b69f20265)
6. Now try to use search field that stays active but not working any more 
7. Even if user click "Done" button and press CMD + F again it will not work either 

Mac OS X 10.11
![screen shot 2015-10-07 at 3 35 23 pm](https://cloud.githubusercontent.com/assets/12688470/10353222/7a6a360a-6d09-11e5-80de-2ed8bc13ec27.png)

**Comment by mmattozzi** [comment]
Yeah, I'm leveraging the search built into https://github.com/faceleg/ACEView and I've noticed the same issues. I'll have to dig into that at some point.
